### ISSUE TITLE
Use show and hide?

### ISSUE THREAD

**Issue by Tymewalk** [Tymewalk/Slither#56]
(Part of #55)

```
def show(self):
    '''Show the sprite.'''
    self.showing = True

def hide(self):
    '''Hide the sprite.'''
    self.showing = False
```

Would it be better to rename `Sprite.showing` to `Sprite.show` and have people manually set it with `True` and `False`?

**Comment by BookOwl** [comment]
Yes.

**Comment by ghost** [comment]
Yes

**Comment by Tymewalk** [comment]
Fixed in #58.
